### PR TITLE
Update register done page

### PR DESCRIPTION
**Pull request by nkarmi**
@waseem-omar @danaspiegel

### CODE REVIEW

**Comment by nkarmi** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;1jqviCAJ&#x2F;1026-1-reg-update-register-done-page-to-include-org-video-cycle-24"> (1) REG - Update register done page to include ORG video [Cycle 24]</a>

File: templates/registration/done.html [hunk 1]
```diff
@@ -46,42 +46,33 @@ <h1 class="title-bold-red">#ENDTHEWAITLIST</h1>
         <div class="col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-24">
             <p class="done-text-bold margin-none">
                 {% blocktrans trimmed %}
-                    Each registered donor can save up to 8 lives.
+                    Spread the word…
```
Comment (danaspiegel) [inline templates/registration/done.html]: check the ellipsis. It shouldn't be an actual ellipsis but rather &hellip;

File: templates/registration/done.html [hunk 2]
```diff
@@ -46,42 +46,33 @@ <h1 class="title-bold-red">#ENDTHEWAITLIST</h1>
...
                 {% endblocktrans %}
             </p>
             <p class="done-text-regular margin-none">
                 {% blocktrans trimmed %}
-                    Telling your friends to register at ORGANIZE is just another way you can help us #EndTheWaitlist
+                    It's important that your friends and family know your donor wishes, so spread the word by tweeting "I am an
```
Comment (danaspiegel) [inline templates/registration/done.html]: please fix the apostrophes and quotes to use html entities

File: templates/registration/done.html [hunk 3]
```diff
@@ -46,42 +46,33 @@ <h1 class="title-bold-red">#ENDTHEWAITLIST</h1>
...
-                <a href="" onclick="fb_share();" class="btn btn-default btn-organize done-btn-social done-btn-share">
-                    {% blocktrans trimmed %}
-                        Share
-                    {% endblocktrans %}
-                </a>
-
-            </div>
+    <div class="row" style="margin: 40px 0;">
+        <div class="col-lg-6 col-lg-offset-3 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-12">
+            <video id="organizeVideo" width="90%" controls="controls">
```
Comment (danaspiegel) [inline templates/registration/done.html]: I think this should be 100%. Is there a reason why you did this? If its necessary, please center the video.

**Pull request merged**